src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -0,0 +1,465 @@
+
+  DevTools highlights the SXG in red because the SXG has errors associated with
+  it. To view these errors and other information about the SXG, click on the SXG,
+  then click **Preview**.
+
+
+
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/MKsFQIvvWrfBj8C3Ce7K.png",
+    alt="Screenshot of the Preview tab showing a SXG with errors",
+    width="800",
+    height="727"
+  %}
+
+  The **Preview** tab displays information about the Signed Exchange and its
+  signature. At the top of the **Preview** tab you should see the error `Failed
+  to fetch certificate`. The browser displays this error when it is unable to
+  load a certificate from the **Certificate URL** indicated in the signature. The
+  next section explains how to fix this error by uploading a certificate.
+
+  Without the certificate, the browser is unable to authenticate the SXG and it
+  falls back to loading the resource without using SXG. This is why there is an
+  additional request to `example.com` listed in the **Network** panel.
+
+
+
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/bU5xNjhvVKr9eSg4RqEZ.png",
+    alt="Screenshot of the DevTools Network tab loading 'example.com' 
+    without using SXG.",
+    width="800",
+    height="192"
+  %}
+
+
+### Upload the self-signed certificate
+
+  To establish the authenticity of a SXG, the browser must be able to load the
+  certificate that was used to sign the SXG from the **Certificate URL** indicated in
+  the signature. If the browser is unable to load this certificate, it will
+  request that the content be delivered without using SXG.
+
+  These instructions explain how to serve a certificate from an existing HTTPS   environment. 
+
+  {% Aside %}
+  `webpkgserver` can be configured to use a locally-hosted
+  certificate. For information on this configuration option,
+  refer to the `CertURLBase` option in `webpkgserver.toml`. This 
+  [article](https://web.dev/how-to-use-local-https/) explains how to 
+  use HTTPS for local development.
+  {% endAside %}
+
+
+#### Prerequisites
+
+  The instructions in this section assume that you have the ability to upload a
+  certificate to an existing HTTPS site. In addition, you should be comfortable
+  adjusting the server configuration of this site.
+
+
+#### Instructions
+
+
+
+1. In DevTools, locate the **Certificate URL** indicated in the
+   **Signature** of the SXG. Copy the hash that is located at the end of this string.
+
+  This hash is an identifier that corresponds to the certificate. If you were to
+  regenerate the SXG using a different certificate, the **Certificate URL**
+  listed in the **Signature** would be different.
+
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/K1caujHPlyCURA1pxYXz.png",
+    alt="Screenshot showing the hash that corresponds to the certificate.",     width="800",
+    height="229"
+  %}
+
+2. Create a copy of `cert.pem`. The filename of the new version should match the
```

Comment:
Seems like the image here breaks the continuity of the numbered list, the steps are rendered as 1, 1, 2, etc